cmd/kpod/images.go

```diff
@@ -155,7 +158,7 @@ func outputImages(store storage.Store, images []storage.Image, format string, ha
 			ID:        img.ID,
 			Name:      name,
 			Digest:    digest,
-			CreatedAt: createdTime,
+			CreatedAt: createdTime.Format("Jan 2, 2006 15:04"),
```

Comment:
That's Go's reference time used for time formats:
https://golang.org/pkg/time/#pkg-constants